qiskit_nature/problems/sampling/protein_folding/resources/mj_matrix.txt

```diff
@@ -0,0 +1,21 @@
+    C       M       F       I       L       V       W       Y       A       G       T       S        N       Q      D       E        H       R       K       P
```

Comment:
Although there is no python file in this module, I'd argue an `__init__.py` would be useful here, simply to be able to document what this matrix is...

Reply:
@pbark Would you please prepare a description of this file that would use right terms for chemists to understand it easily?

Reply:
Done, thanks @pbark